gtsfm/utils/io.py

```diff
@@ -240,13 +237,26 @@ def read_cameras_txt(fpath: str) -> Optional[List[Cal3Bundler]]:
 
         cam_params = line.split()
         # Note that u0 is px, and v0 is py
-        cam_id, model, img_w, img_h, fx, u0, v0 = cam_params[:7]
-        img_w, img_h, fx, u0, v0 = int(img_w), int(img_h), float(fx), float(u0), float(v0)
-
-        # TODO: determine convention for storing/reading radial distortion parameters
-        k1 = 0
-        k2 = 0
-        calibrations.append(Cal3Bundler(fx, k1, k2, u0, v0))
+        model = cam_params[1]
+        # Currently only handles SIMPLE RADIAL and RADIAL camera models
+        assert model in ["SIMPLE_RADIAL", "RADIAL"]
+        if model == "SIMPLE_RADIAL":
+            _, _, img_w, img_h, fx, u0, v0, k1 = cam_params[:8]
+            img_w, img_h, fx, u0, v0, k1 = int(img_w), int(img_h), float(fx), float(u0), float(v0), float(k1)
+            k2 = 0
```

Comment:
Hi Luis, could you also add an inline comment here, saying that if the file contains a COLMAP `SIMPLE_RADIAL` model, we explicitly convert it to GTSAM's "Cal3Bundler" (adding a second radial distortion coefficient of value zero).